sub-packages/bionemo-scdl/src/bionemo/scdl/io/single_cell_collection.py

```diff
@@ -213,24 +213,19 @@ def shape(self) -> Tuple[int, List[int]]:
         return self.number_of_rows(), self.number_of_variables()
 
     def flatten(
-        self,
-        output_path: str,
-        destroy_on_copy: bool = False,
+        self, output_path: str, destroy_on_copy: bool = False, extend_copy_size: int = 10 * 1_024 * 1_024
     ) -> None:
         """Flattens the collection into a single SingleCellMemMapDataset.
 
         Args:
             output_path: location to store new dataset
             destroy_on_copy: Whether to remove the current data_path
```

Comment:
docstring would benefit from an example. If I call `flatten()` on a collection, whats the indented usecase? Should I instantiate a new collection? something else?